packages/grafana-ui/src/components/Table/RowsList.tsx

```diff
@@ -234,8 +239,15 @@ export const RowsList = (props: RowsListProps) => {
       }
 
       if (rowBg) {
-        style.backgroundColor = rowBg(index);
-        style.color = getTextColorForAlphaBackground(style.backgroundColor!, tableStyles.theme.isDark);
+        const { colors, isGradient } = rowBg(index);
+        style.color = colors.textColor;
+        
+        if (isGradient) {
+          style.backgroundImage = colors.bgColor;
+        }
+        else {
+          style.backgroundColor = colors.bgColor;
+        }        
```

Comment:
```suggestion
        const colors = rowBg(index);
        style.color = colors.textColor;
        style.background = colors.bgColor;
```

Reply:
Went for using style.background, definitely more simple 😄